tests/stress/stress_cursor.c

```diff
@@ -973,6 +976,10 @@ main(int argc, char *argv[])
                 para.transaction_count = atoi(optarg);
                 break;
 
+            case 't':
+                para.txn_timeout = atoll(optarg);
+                break;
+
```

Comment:
Wow, I didn't even know atoll() existed.  Probably better to use strtoll(), but I don't see any error checking here so I guess it doesn't matter...